source/ncengine/graphics/NcGraphicsImpl.h

```diff
@@ -31,6 +31,7 @@ class NcGraphicsImpl : public NcGraphics
                        std::unique_ptr<IGraphics> graphics,
                        window::NcWindow& window);
 
+        auto GetApi() const noexcept -> std::string_view override { return "vulkan"; }
```

Comment:
Anywhere we reference the API by a string that needs to match the config value, we should use `graphics::api::Vulkan` (see `NcGraphics.h`). Feel free to remove the `api` namespace there as part of scrubbing the namespaces out of our public include.